src/lightning/fabric/plugins/precision/double.py

```diff
@@ -30,11 +30,14 @@ class DoublePrecision(Precision):
     def convert_module(self, module: Module) -> Module:
         return module.double()
 
-    def init_context(self) -> ContextManager:
```

Comment:
Just to clarify, this is a new feature of 2.1, so it was not yet released as API, correct?

Reply:
Yes, if we do it before 2.1 all good. After 2.1 we wouldn't be able to change the name anymore without deprecation.